WalletWasabi.Gui/Program.cs

```diff
@@ -68,7 +68,7 @@ private static Global CreateGlobal()
 			config.CorrectMixUntilAnonymitySet();
 			var walletManager = new WalletManager(config.Network, dataDir, new WalletDirectories(config.Network, dataDir));
 
-			Logger.InitializeDefaults(Path.Combine(dataDir, "Logs.txt"));
+			Logger.InitializeDefaults(Path.Combine(dataDir, "Logs.txt"), Logging.LogLevel.Trace);
```

Comment:
Undo this, I assume this is why it's draft?

Reply:
Yes, this in the commit "To revert: Turn TRACE logs on for testing." and it is meant to help see what happens when compiled & run.